tools/downloader/common.py

```diff
@@ -36,6 +36,7 @@
     'mxnet': None,
     'pytorch': 'pytorch_to_onnx.py',
     'tf': None,
+    'onnx': None
```

Comment:
Please keep the list in alphabetical order. Also, update the downloader readme and `CONTRIBUTING.md`.